src/core/ext/xds/xds_api.cc

```diff
@@ -187,12 +187,12 @@ void PopulateBuildVersion(const XdsEncodingContext& context,
                           envoy_config_core_v3_Node* node_msg,
                           const std::string& build_version) {
   std::string encoded_build_version = EncodeStringField(5, build_version);
-  // TODO(roth): This should use upb_msg_addunknown(), but that API is
+  // TODO(roth): This should use upb_Message_AddUnknown(), but that API is
```

Comment:
Can this now be changed to use `upb_Message_AddUnknown()`?

Reply:
Good question - @haberman ?

Reply:
So we'll leave this one for a later, smaller change.